Update unions page with more recommendations (#2969)
alan-turing-institute/the-turing-way: book/website/ethical-research/activism/activism-unionisation.md

We recognise that the list below is very UK-centric and would welcome additions for other countries.

#### Global
* Tech Workers Coalition, Global (organisation focused on tech workers in industry) ([website](https://techworkerscoalition.org/))

#### United Kingdom

> * **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or relationship. 

Reviewer: Could you explain what you mean by "relationship" here? Like, employment status at the company (full-time, part-time etc.)?

Author: Yes employment status is what I meant, will update 
```suggestion
* **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or employment status (for example, part-time, full-time, or contractors). 
```

It is an industrial union organised by and for technology workers. ([website](https://utaw.tech/about/))
* **Unite**, United Kingdom, the largest professional services union with over 1 million members ([website](https://unitetheunion.org/))
* **University & Colleges Union (UCU)**, United Kingdom, covers researchers, academic staff and higher grades of professional services staff in Higher Education & Further Education institutions ([website](https://www.ucu.org.uk/))
* **Prospect**, United Kingdom, union covering scientists & engineers ([website](https://members.prospect.org.uk/our-industries/science/index))
* **Public and Commercial Services Union (PCS)**, United Kingdom, civil service union ([website](https://www.pcs.org.uk/about-pcs))
* **United Voices of the World (UVW), Charity Sector Workers Branch** ([website](https://www.uvwunion.org.uk/en/sectors/charity-sector/))
* **Independent Workers’ Union of Great Britain (IWGB), Charity Sector Workers Branch** ([website](https://iwgb.org.uk/page/iwgb-charity-workers-branch/))
    * IWGB also has a number of branches covering cleaners, receptionists and game industry workers

#### Ireland

> * **Medical Laboratory Scientists Association (MLSA)**, Ireland, is the sole trade union representing over 1600 Medical Scientists in Ireland ([website](https://mlsa.ie/who-we-are))

Reviewer: ```suggestion
* **Medical Laboratory Scientists Association (MLSA), Ireland** is the sole trade union representing over 1600 Medical Scientists in Ireland ([website](https://mlsa.ie/who-we-are))
```

Reviewer: I assume this is the Ireland chapter of a bigger organisation?

* **Unione Italiana Lavoratori  (UIL)** is the socialist, social democratic, (republican) and secular split from the CGIL ([website](https://www.uil.it/))
* **Unione Sindacale Professori e Ricercatori (USPUR)** is aimed at university professors and researchers ([website](https://www.uspur.it/))
* **Unione Degli Universitari  (UDU)** is focused on improving the university students experience and future career ([website](https://www.unionedegliuniversitari.it/))

#### Spain
* **Unión General de Trabajadores (UGT)** is a major Spanish trade union, historically affiliated with the Spanish Socialist Workers' Party (PSOE) ([website](https://www.ugt.es/))
* **Confederación Sindical de Comisiones Obreras (CCOO)** is the second largest union in Spain and is the most successful one in labor elections ([website](https://www.ccoo.es/))
* **Federacion de Jovenes Investigadores (FJI-Precarios)** is focused on improving the labour conditions of young researchers, defending the dignity of the research profession, and overall, improving the situation of science in Spain
([website](http://precarios.org/Qui%C3%A9nes+somos))

#### Argentina

> * **APINTA (Asociación del Personal del Instituto Nacional de Tecnología Agropecuaria, Staff Association of the National Institute of Agricultural Technology)** is the entity that groups together all workers, who have an employment relationship with INTA, as well as its retirees and pensioners ([website](https://www.apinta.org/))

Reviewer: ```suggestion
* **APINTA (Asociación del Personal del Instituto Nacional de Tecnología Agropecuaria, Staff Association of the National Institute of Agricultural Technology)** is the entity that groups together all workers who have an employment relationship with INTA, as well as its retirees and pensioners ([website](https://www.apinta.org/))
```
